src/main/java/org/jenkinsci/plugins/parallel_test_executor/SplitStep.java

```diff
@@ -100,7 +121,5 @@ public DescriptorImpl() {
                 return result;
             }
         }
-
```

Comment:
BTW as a rule it is best for functional patches to minimize the diff and so refrain from any formatting changes. Most editors have a setting to only apply a configured formatter to lines which have actually been edited anyway.

Reply:
Got it. :-)
For me it was nearly like a rewrite. Therefore cleaning up anything obvious.